src/bindings/c/include/openvino/c/ov_property.h

```diff
@@ -0,0 +1,150 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief This is header file for ov_property C API.
+ * A header for advanced hardware specific properties for OpenVINO runtime devices.
+ * To use in set_property, compile_model, import_model, get_property methods.
+ * @file ov_property.h
+ */
+
+#pragma once
+
+#include "openvino/c/ov_common.h"
+
+typedef struct ov_property ov_property_t;
```

Comment:
is it a single property? of properties set actually?

Reply:
It is an opaque property handle to represent ov::AnyMap.

Reply:
Why it is named as ov_property? I think ov_any_map or ov_properties, or ov_property_map will be better name

Reply:
I don't thinks so, it is a single property, we name it ov_property is reasonable, and for the multiple property, we can name it as ov_properties_t or ov_property_list_t.